modules/updating-node-pools-for-hcp.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * updates/updating-hosted-control-planes.adoc
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="updating-node-pools-for-hcp_{context}"]
+= Updating node pools for hosted control planes
+
+On hosted control planes, you update your version of {product-title} by updating the node pools. The node pool version must not surpass the hosted control plane version.
+
+.Procedure
+
+* To start the process to update to a new version of {product-title}, change the `spec.release.image` value of the node pool.
+
+.Verification
```

Comment:
Verification steps should be written as single step procedures with a bullet points

Reply:
Will fix ✔️